src/ok-computer.ts

```diff
@@ -303,7 +303,11 @@ export const max = (num: number) =>
     `Expected max ${num}`
   );
 
-export const nullish = err(or(undef, nul), 'Expected nullish');
+export const nullish = err(
```

Comment:
I'm not sure why, but `sideEffects: false` isn't enough and the following PURE annotations are needed also...